platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -330,62 +330,75 @@ metrics:
+  ## Mapping of the event type to the queue name that messages are published to
+  event_type_to_queue:
+    quote_created: ap_event_quote_created
+    transaction_created: ap_event_transaction_created
+    transaction_status_changed: ap_event_transaction_status_changed
+    transaction_error: ap_event_transaction_error
+
   ## @param: publisher_type
-  ## @supported_values: `kafka`, `sqs`
+  ## @supported_values: `kafka`, `sqs`, or `msk`
   ## The type of queue to use for event publishing
   #
-  publisher_type: kafka
-
-
-  options:
-
-    ## @param: bootstrap_server
-    ## @type: string
-    ## The Kafka server used to bootstrap setup
-    ## For MSK, use port 9098 for access from within AWS and port 9198 for public access and specify
-    ## AWS credentials.
-    ## https://docs.aws.amazon.com/msk/latest/developerguide/port-info.html
-    #
-    bootstrap_server: localhost:29092
-
-    ## @param: use_IAM
-    ## @type: boolean
-    ## Use IAM authentication for AWS MSK or AWS SQS.
-    ## SQS FIFO queues should be pre-created in AWS (Anchor Platform will not create them)
-    ## If true, @required_secrets:
-    ##    AWS_ACCESS_KEY_ID
-    ##    AWS_SECRET_ACCESS_KEY
-    #
-    use_IAM: false
-
-    ## @param: region
-    ## @type: string
-    ## AWS region for the queue. Can also be defined as environment variable: AWS_REGION
-    #
-    aws_region: us-east-1
-
-    ## @param: use_single_queue
-    ## @type: boolean
-    ## If true, all events are published to a single queue (specified in eventTypeToQueue.all)
-    use_single_queue: false
-
-    ## @param: event_type_to_queue
-    ## @type: map
-    ## Mapping of the event type to the queue name that messages are published to
-    event_type_to_queue:
-      all: ap_event_single_queue
-      quote_created: ap_event_quote_created
-      transaction_created: ap_event_transaction_created
-      transaction_status_changed: ap_event_transaction_status_changed
-      transaction_error: ap_event_transaction_error
-
+  publisher:
+    type: kafka
+
+    ## If the value of `events.publisher.type` is "kafka", the `events.publisher.kafka` field must be defined.
+    kafka:
+      ## @param: bootstrap_server
+      ## @type: string
+      ## The Kafka server used to bootstrap setup
+      ## For MSK, use port 9098 for access from within AWS and port 9198 for public access and specify
+      ## AWS credentials.
+      ## https://docs.aws.amazon.com/msk/latest/developerguide/port-info.html
+      #
+      bootstrap_server: localhost:29092
+
+
+    ## If the value of `publisher.type` is "sqs", the `events.publisher.sqs` field must be defined.
+    sqs:
+      ## @param: use_IAM
+      ## @type: boolean
+      ## Use IAM authentication for AWS MSK or AWS SQS.
+      ## SQS FIFO queues should be pre-created in AWS (Anchor Platform will not create them)
+      ## If true, @required_secrets:
+      ##    secret.event.publisher.aws_access_key_id
+      ##    secret.event.publisher.aws_secret_access_key
```

Comment:
Should we make secrets uppercase?

Reply:
Also, will you copy the values from these envs into `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`?

Reply:
The secret env vars are case insensitive. This is to make it consistent with the `secret.sep10.signing_seed` env var name.

And yes, they will be copied to the config class and set in the Java code.

Reply:
What I mean is these env vars (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`) are picked automatically by the aws-msk libs. I don't know if this is something we can inject into the objects using the Java code.

Reply:
@marcelosalloum, @stfung77 I do not have answers either. But I think they can be figured out and document when we implement. `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are part of secret management. We can figure out after the schema change.

Reply:
SGTM!

Reply:
Document updated as the following: 
```yaml
      ## If true, please follow the `Set up AWS Credentials and Region for Development`
      ## at https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html
      #
```